cluster/packages.sls

```diff
@@ -1,6 +1,7 @@
 #required packages to install HA cluster
 
 {%- from "cluster/map.jinja" import cluster with context -%}
+{% set provider = pillar.cluster.configure.template.parameters.platform %}
```

Comment:
2 things:
- I wouldn't put `pillar` here. Just start with `cluster` as this way the data is merged with `defaults` and any other change done in `map.jinja`
- What happens if this entry doesn't exist? Do we need some default value? I think it would fail, right? Consider that the formula might be used with those entries, they are optional

Reply:
please, upgrade package version and changes file!!!